www/_includes/header.html

```diff
@@ -24,6 +24,7 @@
 				<a {% if page.url == "/kontakt.html"                  %} class="active" {% endif %} href="/kontakt.html">Kontakt</a>
 				<a data-no-turbolink href="https://www.noname-ev.de/w/">Wiki</a>
 				<a {% if page.url == "/cryptoparty.html"              %} class="active" {% endif %} href="/cryptoparty.html">Cryptoparty</a>
+				<a {% if page.url == "/english.html"                  %} class="active" {% endif %} href="/english.html" lang="en">English</a>
```

Comment:
Too much effort, but feel free to give it a try: https://stackoverflow.com/questions/14920401/how-to-simulate-hfill-with-html-and-css The most obvious `float: right` fails once the menu goes into overflow mode because the element gets pushed down.